src/lib/containers/UserCardList.tsx

```diff
@@ -96,7 +96,7 @@ export default class UserCardList extends React.Component<
         lastName: values[lastNameIndex],
         company: values[institutionIndex],
         ownerId: '',
-        userName: values[firstNameIndex][0],
+        userName: '',        
```

Comment:
This is a bug that I need to fix.  Now I realize this is to support the default picture (first letter of the username), but it needs to handle when the first name is not set.